gempython/tests/getVFATMask.py

```diff
@@ -0,0 +1,19 @@
+#!/bin/env python                                                                                                                                                                                                                       
```

Comment:
This (file) shouldn't be here, the PR introducing it was explicitly done against a specific, short-term branch